src/main/java/org/cactoos/iterator/IteratorOf.java

```diff
@@ -23,64 +23,59 @@
  */
 package org.cactoos.iterator;
 
-import java.util.Collection;
 import java.util.Iterator;
-import java.util.LinkedList;
-import org.cactoos.iterable.IterableOf;
-import org.cactoos.scalar.StickyScalar;
-import org.cactoos.scalar.UncheckedScalar;
+import java.util.NoSuchElementException;
+import java.util.concurrent.atomic.AtomicInteger;
 
 /**
- * Iterator that returns the same set of elements always.
+ * Iterator that returns the set of elements.
  *
- * <p>There is no thread-safety guarantee.
+ * <p>There is no thread-safety guarantee.</p>
  *
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @param <X> Type of item
  * @since 0.8
```

Comment:
@Maratori here it should be ``@since 0.30``, since it's a new class now

Reply:
done